fitparser/src/de/decode.rs

```diff
@@ -14,6 +14,7 @@ use std::convert::{From, TryInto};
 pub struct Decoder {
     base_timestamp: TimestampField,
     accumulate_fields: HashMap<u32, Value>,
+    pub developer_fields: HashMap<(u8, u8), DeveloperFieldDescription>,
```

Comment:
I do wonder if we'd want to put this behind a getter since it's passed around outside of the immediate context but it's not actually publicly exposed outside of the `de` module either.

Reply:
I put it behind a getter.